### DISCUSSION TITLE
While Saving annotation program crash

### DISCUSSION THREAD

**Discussion by nOvlc3** [sumatrapdfreader/sumatrapdf#3477] [Q&A] [answered]
# sumatrapdf crash while saving annotation (Intentionally)[^1]

## Environment
- [x] Win11
- [x] Latest Version (SumatraPDF-3.4.6-64)
---

I use mouse selecting a block of contents, and then right click it, choosing "Create Annotation From Selection" with "Highlight". After finish all the configuration like selecting color or writing annotations, I left click **Save changes to existing PDF**, but twice. Then I can't operate it any more and it crash soon (few seconds).

The result of crash is that the annotation is **created correctly**. All the book I opened have restored. So the problem **DOESN'T** bother me seriously.

[^1]As I write above *Intentionally*, this problem won't happen if I just click once and wait for a few seconds. So if you don't mind it , just regrading what I say as trash. Hoping this doesn't interrupt your life. Anyway, thank you for who read here or think my problem. 

---
By the way, I not a English-native speaker. And the markdown I write maybe not formal. Any suggestion will be a reward for me.

**Comment by GitHubRulesOK** [comment, marked as answer]
There were some cases when SumatraPDF 3.4.6 would crash
Many have been fixed in the current Pre-Release
https://www.sumatrapdfreader.org/prerelease
Only Pre-Release can be updated if there are any repeatable crashes

**Reply by nOvlc3** [reply]
# New Version Work Correctly

#### I **download Pre-Release** and repeat what I did for previous version (lastest release) and it **doesn't crash**.
---
Thank for your reply. Samatra pdf is the best pdf reader. I will keep on supporting it.
